GameData/KerbalismConfig/Support/EngineIgnitor.cfg

```diff
@@ -1,5 +1,5 @@
 // disable all engine failures
-@PART[*]:HAS[@MODULE[ModuleEngines*]]:NEEDS[EngineIgnitor]:AFTER[Kerbalism]
```

Comment:
KerbalismDefault will disappear with Kerbalism 4.0 (which the dev branch this patch is against)